free-1.md

```diff
@@ -5,47 +5,63 @@
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+accessable langauge for quick/affective Information storage
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+My though would be that the language would be a combination of a natural language and an optional coding like langauge. For instance users that want to use this aplication for personal knowlegde such as books, reports, websites ect... would have thia ablity to use either a phone or computer to efforlessly say something like " store book "title of book" "anything else they deem important about the book such as location in collection" " where the language would use precise key words to do an action. Additionally, mayble this language presents itself more useful for database programers that have to phisicallycode the database with the type of information to be stored, how much space to use, or other data base process that have to be completed each time you build a database. In this example we could have a DSL that still works roughly the same as above but may allow for more coding ability such as add, recover, delete, and edit funcftions. This would allow for programers to get rid of most of their set up and hard coding of the data base and move into just the what do I want to do with it phase. 
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+lets say the user wants to store information they may use a key word such as store or save ... The laguange would see the keyword and automaticcly go through the process of setting up a data base. Lets also say they wanted to save a book and maybe we have a keyword for book, this would allow the system to start a category within the dabase for books with additional automation. once the users request is put through it would comunicate back with a success or fail message where the fail message may be because of an operational issue or maybe becuase the language used by the user did not mach the specifations of the DSL.
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy for the user to put, gather, and edit info within a database through a limited key word language. This shoud be possible but may be difficult to program effectivly. I dont believe anything within this project would be impossible but the natural langauge component to actual process in real time with automation may be more difficult. In the sence that when a keyword is detected the program is basically writing a file for a database auomaticaly instead of the user or programmer. 
+
```

Comment:
From my understanding, it would be difficult to make more fine-grained edits of the database itself. Is that something that you would want some users to be able to do? Like, beyond the natural language component? or would it be hidden unless a more experienced user wants to.